src/App.js

```diff
@@ -85,6 +85,8 @@ function App() {
 
   const [isErrorEnabled, setIsErrorEnabled] = useState(false)
   const [isMFAEnabled, setIsMFAEnabled] = useState(false)
+  const [isStringResponseToSend, setIsStringResponseToSend] = useState(false)
+  const [isCancelButtonEnabled, setIsCancelButtonEnabled] = useState(false)
```

Comment:
what is the `cancelButton` for?

Reply:
In PointServe Figma. they have a cancel button. TTO wants that functionality so that users can cancel the process anytime. It will be done through the widget side to go back to the original screen.